scripts/update_schema.py

```diff
@@ -92,16 +91,13 @@ def update_schema(sublime_package_json: JsonDict, pyrightconfig_schema_json: Jso
     # then it might have to be added manually.
     all_settings_keys = [key.rpartition(".")[2] for key in settings_properties.keys()]
     all_overrides_keys = settings_properties["python.analysis.diagnosticSeverityOverrides"]["properties"].keys()
-
-    yield from (
-        pyrightconfig_key
-        for pyrightconfig_key in pyrightconfig_properties.keys()
-        if not (
-            pyrightconfig_key in all_settings_keys
-            or pyrightconfig_key in all_overrides_keys
-            or pyrightconfig_key in IGNORED_PYRIGHTCONFIG_KEYS
-        )
-    )
+    new_schema_keys = []
+    for pyrightconfig_key in pyrightconfig_properties.keys():
+        if pyrightconfig_key not in all_settings_keys \
+                and pyrightconfig_key not in all_overrides_keys \
+                and pyrightconfig_key not in IGNORED_PYRIGHTCONFIG_KEYS:
+            new_schema_keys.append(pyrightconfig_key)
+    return new_schema_keys
```

Comment:
Sorry for reverting this but it seemed like an over-engineering. The main benefit of generators is that the caller can break the loop at any point. In this case we're collecting and returning all keys so I don't see the point.
Even if performance doesn't matter here, I don't like  over complicating things.